packages/main/src/TableVirtualizer.ts

```diff
@@ -170,14 +170,18 @@ class TableVirtualizer extends UI5Element implements ITableFeature {
 	}
 
 	get _scrollContainer() {
-		return this._table!._tableElement;
+		return this._table?._tableElement;
 	}
 
 	get _rowsContainer() {
 		return this._table!.shadowRoot!.getElementById("rows")!;
 	}
 
 	_onScroll(): void {
+		if (!this._scrollContainer) {
```

Comment:
scroll is a sync event, how the scroll container that fires the scroll event might not exists? yes we call it from onAfterRendering but as i mentioned there the shadow dom of the table must exist in the onAfterRendering of the table.